site/content/installation/installing-nic/deploy-with-nap-using-helm.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Deploy NIC and WAF to Kubernetes using Helm
+toc: true
+weight: 500
+type: how-to
+product: NIC
+docs: DOCS-609
+---
+
+## How To: Build and Use a Local NGINX App Protect Docker Image to Compile WAF Policies and Integrate with NGINX Plus Ingress Controller
+
+This document outlines how to:
+
+- Set up license secrets to enable deployment into kubernetes
+- Use a WAF docker image to compile a policy JSON file into a compiled bundle
+- Set up PersistentVolumes that the deployed WAF instance can access to read the bundle from
+- Deploy NGINX Plus Ingress Controller with NAP in Kubernetes
+- Test example services to validate that the WAF policies work
+
+---
+
+## Step 1: Prepare Secrets and Credentials (From MyF5 Portal)
+
+Download the following files from your [MyF5 Portal](https://my.f5.com):
+
+- `nginx-repo.crt`
+- `nginx-repo.key`
+- `nginx-jwt.token`
+
+Store these files locally:
+
+```
+├── nginx-repo.crt
+├── nginx-repo.key
+└── nginx-repo.jwt
+```
```

Comment:
We have some examples of this kind of instruction you can likely copy regarding license files: they exist as `include` files, which are inaccessible until the documentation is merged into the wider documentation repository.

https://docs.nginx.com/nginx-instance-manager/deploy/docker/deploy-nginx-instance-manager-docker-compose/#set-up-docker-for-nginx-container-registry

https://github.com/nginx/documentation/blob/main/content/nim/deploy/docker/deploy-nginx-instance-manager-docker-compose.md

https://github.com/nginx/documentation/blob/main/content/includes/nim/docker/docker-registry-login.md

Reply:
NIC doesn't have the necessary include for this, and the includes only concern themselves with the jwt tokens. For this piece of documentation I need all three files.

Reply:
I know: I'm mostly pointing to them as an example of a way to phrase the instructions.

We aim to try to have as similar a reading experience as possible for users across the whole NGINX ecosystem, so it's less about re-using this specific text and looking at how the text is structured.

You can copy the list text and structure nearly verbatim, updating step 4 to include the UI name for each of the downloaded files.

The current text just says "go here to download the files" without any guidance on where in MyF5 to get them.